conf/config-default.yaml

```diff
@@ -174,6 +174,11 @@ nginx_config:                     # config for render the template to generate n
   #  - TEST_ENV
 
   stream:
+    enable_access_log: true         # enable access log or not, default true
+    access_log: logs/access_stream.log
+    access_log_format: "$remote_addr [$time_local] $status $bytes_sent $session_time $upstream_connect_time \"$protocol://$upstream_addr\""
```

Comment:
@spacewander by consulting http://nginx.org/en/docs/varindex.html, I created the log format. Do you think we can do better? Do let me know. Thanks.

Reply:
I think we can use Nginx's basic format in its example?
```
log_format basic '$remote_addr [$time_local] '
                 '$protocol $status $bytes_sent $bytes_received '
                 '$session_time';
```
See https://nginx.org/en/docs/stream/ngx_stream_log_module.html.

Kong also uses it too: https://github.com/Kong/kong/blob/d65101fe80fd7ac9870a84d34d81bda8bcb461ac/kong/templates/nginx_kong_stream.lua#L3

Reply:
Okay.